### PR TITLE
Zachran seba, zachranis svet

### PR DESCRIPTION
**Pull request by urbanga**
Please review this, @casey-collab

### CODE REVIEW

File: slovak/zachrana/zachrana-seba.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rybku rýchlo vytiahneš z nosa.
```
Comment (casey-collab) [inline slovak/zachrana/zachrana-seba.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by urbanga** [comment]
Please review this, @casey-collab

File: slovak/marsmelaci.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ Jedno nevšedné ráno sa zobudíš v divnej miestnosti, ktorá má steny z mars
 
 [Pokúsiš sa zachrániť rybku](zachrana/zachrana-rybky.md)
 
+[Pokúsiš sa zachrániť seba](zachrana/zachrana-seba.md)
```
Comment (casey-collab) [inline slovak/marsmelaci.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: slovak/nepi/zachrana-seba.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Rybku rýchlo vytiahneš z nosa.
```
Comment (casey-collab) [inline slovak/nepi/zachrana-seba.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by urbanga** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**